tests/add_employee.py

```diff
@@ -0,0 +1,72 @@
+# Homework:
+# 1: login
+# 2: click Add button, to add a new employee
+# 3: enter required data
+# 4: save employee id
+# 5: click save
+# 6: ensure new employee was created
+# 7: perform search for the employee by the employee id, and ensure all data matches.
+# Use WebDriverWait as needed
+import unittest
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+
+from fixtures.base_fixture import BaseFixture
+from tests.search_employee import get_greeting
+
+
+def get_greeting(browser):
+    return browser.find_element(By.ID, 'welcome').text
+
+
+def logout(browser):
+    current_url = browser.current_url
+    browser.find_element(By.ID, 'welcome').click()
+    wait = WebDriverWait(browser, 3)
+    wait.until(
+        expected_conditions.visibility_of_element_located(
+            (By.LINK_TEXT, 'Logout'))).click()
+    wait.until(expected_conditions.url_changes(current_url),
+               'url failed to change after logout was clicked')
+
+
+class MyTestCase(BaseFixture):
```

Comment:
Probably best to change the class name ;)

Also, it may be more convenient to use the AdminLoginFixture here